src/Tulsi/HeadlessTulsiProjectCreator.swift

```diff
@@ -193,6 +195,9 @@ struct HeadlessTulsiProjectCreator {
 
     // Add a single source path including every possible source.
     configDocument.sourcePaths = [UISourcePath(path: ".", selected: true, recursive: true)]
+    if additionalSourcePaths != nil {
+        configDocument.sourcePaths += additionalSourcePaths!.map { UISourcePath(path: $0, selected: true, recursive: true) }
```

Comment:
I suspect you may want recursive: true but selected: false. Otherwise you may get a duplicate entry.